docs/workload-identity-static-pv-mount.md

```diff
@@ -0,0 +1,197 @@
+# Example of static PV mount with workload identity
+
+> Note: 
+> - Available kubernetes version >= v1.20
+
+## prerequisite
+
+### 1. Create a cluster with oidc-issuer enabled
```

Comment:
there is already doc here: https://github.com/kubernetes-sigs/azurefile-csi-driver/blob/master/docs/workload-identity.md, it's better removing redundant doc steps otherwise we need to maintain two doc pages.

Reply:
OK

Reply:
Basically, these are two very different features for different purpose although they both use federated workload identity and actually there are very few common steps between these two docs and maintain two docs should be better.